flink-table/flink-table-planner/pom.xml

```diff
@@ -22,13 +22,18 @@ under the License.
 
 	<parent>
 		<groupId>org.apache.flink</groupId>
-		<artifactId>flink-libraries</artifactId>
+		<artifactId>flink-table</artifactId>
 		<version>1.8-SNAPSHOT</version>
 		<relativePath>..</relativePath>
 	</parent>
 
-	<artifactId>flink-table_${scala.binary.version}</artifactId>
-	<name>flink-table</name>
+	<artifactId>flink-table-planner_${scala.binary.version}</artifactId>
+	<name>flink-table-planner</name>
+	<description>
+		This module bridges Table/SQL API and runtime. It contains
+		all resources that are required during pre-flight and runtime
+		phase.
+	</description>
```

Comment:
Could we remove `flink-table-common` dependency in this file? It has already been included by `flink-table-api-java-bridge`.

Reply:
Initially, I didn't want to add transitive dependencies. But @aljoscha pointed out that we should declare dependencies if we use those classes. See also:
https://stackoverflow.com/questions/15177661/maven-transitive-dependencies

Reply:
Make sense. Thanks for the explanation.